SimG4CMS/HcalTestBeam/plugins/HcalTB06Analysis.cc

```diff
@@ -11,8 +11,8 @@
 //
 
 // user include files
-#include "HcalTB06Histo.h"
-#include "HcalTB06BeamSD.h"
+#include "SimG4CMS/HcalTestBeam/plugins/HcalTB06Histo.h"
```

Comment:
@bsunanda , this modification is fine in general but is it really needed or it kind of cosmetic change?

Reply:
Indeed, all these should be better reverted: inclusions from the same plugin directory should remain without indicating the full address (in order to avoid that in case someone move or copy this code there are inclusions from a different plugin directory)

Reply:
I am reverting them